.nycrc.json

```
@@ -1,5 +1,5 @@
 {
     "all": true,
     "include": ["src/**", "dist/src/**"],
-    "exclude": ["test/**", "integrationTest/**", "testFixtures/**"]
+    "exclude": ["test/**", "testE2E/**", "testLint/**", "testInteg/**", "testFixtures/**"]
```

should these be prefixed with `src/` ?
```suggestion
    "exclude": ["src/test/**", "src/testE2E/**", "src/testLint/**", "src/testInteg/**", "src/testFixtures/**"]
```

---

I think the patterns are applied relative to `include` but not 100% sure

---

that seems to be what the examples in https://github.com/istanbuljs/nyc#selecting-files-for-coverage imply, but it doesn't clearly say.